### PR TITLE
feat(core): add connector config examples

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Add connector config examples.

## Linear Issue Reference

LOG-2150

## Testing

UTs.

### CODE REVIEW

File: packages/core/src/connectors/alipay/config-template.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 {
-  // placeholder
+  "appId": "2021000000000000",
```
Comment (demonzoo) [inline packages/core/src/connectors/alipay/config-template.md:2]: Use a placeholder instead?

File: packages/core/src/connectors/aliyun-dm/config-template.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
 {
-  // placeholder
+  "accessKeyId": "<your-access-key-id>",
+  "accessKeySecret": "<your-access-key-secret>",
+  "accountName": "no-reply@example.com",
+  "fromAlias": "ExampleSite",
```
Comment (demonzoo) [inline packages/core/src/connectors/aliyun-dm/config-template.md:5]: Better wrap anything the user needs to replace in a pair of angle brackets

File: packages/core/src/connectors/alipay/config-template.md (hunk 1 above)
Comment (darcyYe) [reply packages/core/src/connectors/alipay/config-template.md]: fixed.

File: packages/core/src/connectors/aliyun-dm/config-template.md (hunk 2 above)
Comment (darcyYe) [reply packages/core/src/connectors/aliyun-dm/config-template.md]: fixed.

**Review by demonzoo — APPROVED** [review]

**Pull request merged**